elao.app/MIGRATION.2022-01.md

```diff
@@ -0,0 +1,178 @@
+# Migration - 2022-01
+
+:warning: Launch a 'make destroy' under current code base before anything!
+
+:warning: Read carefully README's requirements!
+
+## Project
+
+Replace system.hostname by project.name in `.manala.yaml`, without forgetting to remove the `.vm` extension!
+
+```
+###########
+# Project #
+###########
+
+project:
+    name: foo-bar  # <-- Yep! That's it!
+
+...
+
+system:
+    ...
+    hostname: foo-bar.vm  # <-- So 2021...
+```
```

Comment:
```suggestion
```diff
###########
# Project #
###########

-system:
+project:
-    hostname: foo-bar.vm  # <-- So 2021...
+    name: foo-bar  # <-- Yep! That's it!
```
```

Reply:
Plus mitigé dans ce cas, ça donne l'impression qu'on peut gicler toute la clé `system` alors que non, c'est juste `system.hostname`